qiskit/transpiler/target.py

```diff
@@ -1311,151 +1314,218 @@ def from_configuration(
                             try:
-                                duration = backend_properties.gate_length(gate, qubit)
-                            except BackendPropertyError:
-                                duration = None
-                        try:
-                            error = backend_properties.gate_error(gate, qubit)
-                        except BackendPropertyError:
-                            error = None
-                    if inst_map is not None:
-                        try:
-                            calibration = inst_map._get_calibration_entry(gate, qubit)
-                            # If we have dt defined and there is a custom calibration which is user
-                            # generate use that custom pulse schedule for the duration. If it is
-                            # not user generated than we assume it's the same duration as what is
-                            # defined in the backend properties
-                            if dt and calibration.user_provided:
-                                duration = calibration.get_schedule().duration * dt
-                        except PulseError:
-                            calibration = None
-                    # Durations if specified manually should override model objects
-                    if instruction_durations is not None:
-                        try:
-                            duration = instruction_durations.get(gate, qubit, unit="s")
-                        except TranspilerError:
-                            duration = None
+                                # Qubits might be pre-defined by the gate config
+                                # However properties objects says the qubits is non-operational
+                                del prop_name_map[name][qubits]
+                            except KeyError:
+                                pass
+                            faulty_ops.add((name, qubits))
+                            continue
 
-                    if error is None and duration is None and calibration is None:
-                        gate_properties[(qubit,)] = None
-                    else:
-                        gate_properties[(qubit,)] = InstructionProperties(
-                            duration=duration, error=error, calibration=calibration
-                        )
-                target.add_instruction(name_mapping[gate], properties=gate_properties, name=gate)
-            edges = list(coupling_map.get_edges())
-            for gate in two_qubit_gates:
-                gate_properties = {}
-                for edge in edges:
-                    error = None
-                    duration = None
-                    calibration = None
-                    if backend_properties is not None:
-                        if duration is None:
+                        error = params["gate_error"][0] if "gate_error" in params else None
+                        if instruction_durations is not None:
```

Comment:
According to the docstring you must cover the case that the `backend_properties` is None but the `instruction_durations` exists.

Reply:
I was not sure if the docstrings should be changed or the logic itself should be changed. That's why I request your guidance :)
Applied suggestion :)